Source/Common/Common.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Ivi.Driver;
+
+namespace NationalInstruments.ReferenceDesignLibraries
+{
+    public enum LocalOscillatorSharingMode
+    {
+        None,
+        Automatic,
+        Manual
+    }
+
+    public enum LocalOscillatorOffsetMode
+    {
+        NoOffset,
+        Automatic,
+        UserDefined
+    }
+
+    public struct LocalOscillatorConfiguration
+    {
+        public string ChannelName;
+        public LocalOscillatorSharingMode SharingMode;
+        public string Source;
+        public bool ExportEnabled;

```

Comment:
Where did we land on this? Are you still motivated to change the name? 
I'd challenge that "Manual" could also apply to properties in other modules as well. For example, SweepTimeAuto. Should we have ManualSweepTime_s as a property instead of SweepTime_s?

Reply:
Good point! Let's leave it alone.